air/src/air/mod.rs

```diff
@@ -188,6 +192,12 @@ pub trait Air: Send + Sync {
     /// This could be any type as long as it can be serialized into a sequence of field elements.
     type PublicInputs: ToElements<Self::BaseField>;
 
+    /// An auxiliary (i.e. non-STARK) proof object. If not needed, set to `()`.
+    type GkrProof: Serializable + Deserializable;
+
+    /// An auxiliary (i.e., non-STARK) verifier for verifying GKR proofs. If not needed, set to [`DefaultGkrVerifier`].
+    type GkrVerifier: GkrVerifier<GkrProof = Self::GkrProof>;
```

Comment:
minor nit: should the comments now mention GKR specifically? Also, we probably don't need a reference to the `DefaultGkrVerifier`.

Reply:
Oops, forgot those. Fixed